website/content/api-docs/secret/transform.mdx

```diff
@@ -1048,16 +1066,20 @@ $ curl \
 
 This endpoint decodes the provided value using a named role.
 
-| Method | Path                           |
-| :----- | :----------------------------- |
-| `POST` | `/transform/decode/:role_name` |
+| Method | Path                                            |
+| :----- | :---------------------------------------------- |
+| `POST` | `/transform/decode/:role_name(/:decode_format)` |
 
 ### Parameters
 
 - `role_name` `(string: <required>)` –
   Specifies the role name to use for this operation. This is specified as part
   of the URL.
 
+- `decode_format` `(string)` -
```

Comment:
Should this not be after value since decode_format is an optional vs value being a required argument? 

Or is this due to this parameter being a path param and value being within the body?